pkg/controllers/namespace.go

```diff
@@ -22,7 +22,7 @@ import (
 	"sync"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I don't know why we need 'v1'... Could you please clarify why we need it?

Reply:
hmm... interesting. I surprised that gofmt enforces to users golang unified format but also gopls (language server) enforces to us to some format...

Understand that, so let's change to 'v1 "k8s.io/api/core/v1"' for now.